shell/common/shell_unittests.cc

```diff
@@ -597,201 +597,6 @@ TEST_F(ShellTest,
   DestroyShell(std::move(shell));
 }
 
-TEST_F(ShellTest, OnPlatformViewDestroyAfterMergingThreads) {
```

Comment:
Ideally, this test should pass with this change.

Reply:
Done.